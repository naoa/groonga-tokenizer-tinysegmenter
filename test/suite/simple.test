register tokenizers/tinysegmenter

tokenize TokenTinySegmenter "今日は雨です。"
