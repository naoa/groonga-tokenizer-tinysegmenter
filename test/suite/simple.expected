register tokenizers/tinysegmenter
[[0,0.0,0.0],true]
tokenize TokenTinySegmenter "今日は雨です。"
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "今日",
      "position": 0
    },
    {
      "value": "は",
      "position": 1
    },
    {
      "value": "雨",
      "position": 2
    },
    {
      "value": "です",
      "position": 3
    },
    {
      "value": "。",
      "position": 4
    }
  ]
]
